pkg/operator/ceph/csi/controller.go

```diff
@@ -73,7 +74,7 @@ func newReconciler(mgr manager.Manager, context *clusterd.Context, opManagerCont
 		context:          context,
 		opConfig:         opConfig,
 		opManagerContext: opManagerContext,
-		multusClusters:   []ClusterDetail{},
+		clusters:         []ClusterDetail{},
```

Comment:
For now these are only the clusters that use the holder pod, right? In the future refactor PR, then we can rename it to `clusters` if we collect all clusters.
```suggestion
		holderClusters:         []ClusterDetail{},
```